app/src/main/AndroidManifest.xml

```diff
@@ -14,9 +14,8 @@
     <activity android:name="org.oppia.app.player.state.testing.StateFragmentTestActivity"/>
     <activity android:name=".player.exploration.ExplorationActivity"/>
     <activity android:name=".topic.questionplayer.QuestionPlayerActivity"/>
-    <activity android:name=".topic.TopicActivity"/>
-    <activity android:name="org.oppia.app.player.state.testing.StateFragmentTestActivity"/>
-    <activity android:name="org.oppia.app.home.HomeActivity">
+    <activity android:name=".topic.TopicActivity"
+              android:theme="@style/OppiaTheme.NoActionBar">
       <intent-filter>
```

Comment:
`HomeActivity` should be the launcher activity and not `TopicActivity` because if this PR gets merged than again we will have to make one more PR to make launcher activity back to `HomeActivity` which would be incorrect.

I suggest in such cases keep the natural flow of apps intact and in PR description write steps on how to check/run this PR.

Reply:
+1 though feel free to add a button or something in the home fragment to easily route to the topic activity to make things easier.

Reply:
resolved